### DISCUSSION TITLE
Prevent cypress from accessing api-proxy.cypress.io

### DISCUSSION THREAD

**Discussion by maximilian-krauss** [sorry-cypress/cy2#75] [Q&A] [answered]
Hi!

We are trying to setup sorry cypress in our build environment. The build runner is not able to talk to the internet and we discovered that cypress is trying to reach:

```shell
cy2-net Connection request: api-proxy.cypress.io:443
```

This fails obviously and the whole run stops because that endpoint is not accessible. Do you have any idea how we can disable those calls to cypress?

Thanks!

Cheers,
Max

**Comment by agoldis** [comment, marked as answer]
Please use https://github.com/currents-dev/cypress-cloud instead

**Reply by maximilian-krauss** [reply]
Hey,
can I host currents inhouse? Otherwise I would have the same problem again because it needs to communicate with the currents cloud

**Reply by agoldis** [reply]
The package is compatible with sorry cypress and doesn't need anything else (expect from cypress runner) - i.e. it won't connect to external services

**Reply by maximilian-krauss** [reply]
Thank you very much, that did the trick! 💚
